packages/data-table-server/src/dataTableService/services/DataElementMetadataDataTableService.ts

```diff
@@ -10,7 +10,7 @@ import { yup } from '@tupaia/utils';
 import { DataTableService } from '../DataTableService';
 
 const requiredParamsSchema = yup.object().shape({
-  dataElementCodes: yup.array().of(yup.string().required()).required(),
+  dataElementCodes: yup.array().of(yup.string().required()).default([]),
```

Comment:
Test fail shows this is actually meant to be required?

Reply:
I suggest we remove the unit test and make it accept dataElementCodes as undefined.

Reply:
Yep sounds good